src/pages/sok.tsx

```diff
@@ -0,0 +1,213 @@
+import { useLocale } from "@/locales";
+import { useState, useRef } from "react";
+function SingleEvent({
+  bgColor,
+  borderColor,
+  toReverse,
+  textColor,
+  image,
+  eventInfo,
+  price,
+  roles,
+}: {
+  bgColor: string;
+  borderColor: string;
+  toReverse: boolean;
+  textColor: string;
+  image: string;
+  eventInfo: string[];
+  price: string;
+  roles: string[];
+}) {
+  const t = useLocale();
+  const [modalState, setModal] = useState(false);
+  const modalRef = useRef<HTMLDivElement>(null);
+
+  const openModal = () => {
+    setModal(true);
+  };
+
+  const closeModal = () => {
+    setModal(false);
+  };
+
+  const handleOverlayClick = (event: React.MouseEvent) => {
+    if (modalRef.current === event.target) {
+      closeModal();
+    }
+  };
+
+  return (
+    <div
+      className={`${
+        toReverse
+          ? "md:flex-row-reverse flex-col-reverse"
+          : "md:flex-row flex-col-reverse"
+      } flex flex-row gap-[50px] px-[50px] md:px-[50px] mt-[100px] justify-center`}
+    >
+      <div className="py-[0px]">
+        <img src={image} className="md:h-[300px] lg:h-[300px] rounded-xl"></img>
+      </div>
+      <div
+        className={`${bgColor} ${borderColor} border-[3px] rounded-lg py-[25px] md:py-[25px] lg:py-[50px] px-[20px] bg-opacity-10 md:w-[300px] md:h-[300px] lg:w-[400px] lg:h-[300px]`}
+      >
+        <h2 className="text-center lg:text-3xl md: text-xl text-white">
+          {eventInfo[0]}
+        </h2>
+        <h3 className="text-center text-cerise  lg:text-xl md:mt-1 lg:mt-2"></h3>
+        <p className="text-white text-start mt-5 text-sm">{eventInfo[1]}</p>
+        <button className={`${textColor} text-start mt-3`} onClick={openModal}>
+          <u>{t.sok.extra}</u>
+        </button>
+
+        {/*MODAL*/}
+        {modalState && (
+          <div
+            className="fixed inset-0 flex items-center justify-center bg-black bg-opacity-60 backdrop-blur-lg"
+            ref={modalRef}
+            onClick={handleOverlayClick}
+          >
+            <div
+              className={`bg-white bg-opacity-70 w-[500px] max-h-[80vh] overflow-y-auto pb-5 flex flex-col rounded-3xl`}
+              style={{ scrollbarWidth: "none", msOverflowStyle: "none" }}
```

Comment:
I can still see the scrollbar. Do we want it?
<img width="601" alt="image" src="https://github.com/datasektionen/ddagen/assets/66337956/4ec795ef-6e02-4282-87ae-73015e7d09a2">